### ISSUE TITLE
@html-eslint/require-lang does not work for lang attributes with Country Codes

### ISSUE THREAD

**Issue by ricealexander** [yeonjuan/html-eslint#29]
The lang attribute can accept Country Codes in addition to the language, to more precisely identify how the page content is written. `@html-eslint/require-lang` does not recognize lang attributes with [Country Codes](https://hexagonitsolutions.com/ngo/ref_country_codes.php).

I am using the recommended configuration:
<details>
<summary>.eslintrc.js</summary>
<br>

```js
module.exports = {
  plugins: ['@html-eslint'],
  overrides: [
    {
      files: ['*.html'],
      parser: '@html-eslint/parser',
      'extends': ['plugin:@html-eslint/recommended'],
    },
  ],
}
```

</details>

The following code demonstrates the issue:

```html
<!DOCTYPE html>
<html lang="en-US">
    <body>
        <h1>Page Title</h1>
    </body>
</html>
```
.
This provides an error of "**Invalid value of \`lang\` attribute**" even though it is valid.

Removing the "-US" removes the error.

**Comment by yeonjuan** [comment]
@ricealexander  Thanks for the report :) I realized that the correct validation of 'lang'  quite needs hard work. I changed it to just report an empty value. see [demo](http://localhost:3000/html-eslint/playground#eyJjb2RlIjoiPCFET0NUWVBFIGh0bWw+XG48aHRtbCBsYW5nPVwiZW4tVVNcIj5cbjwvaHRtbD5cbiIsInJ1bGVzIjp7IkBodG1sLWVzbGludC9pbmRlbnQiOiJlcnJvciIsIkBodG1sLWVzbGludC9yZXF1aXJlLWxpLWNvbnRhaW5lciI6ImVycm9yIiwiQGh0bWwtZXNsaW50L3JlcXVpcmUtbGFuZyI6ImVycm9yIn19).
